tensorflow/compiler/xla/service/copy_insertion_test.cc

```diff
@@ -2464,7 +2466,6 @@ ENTRY TestComputation {
   TF_ASSERT_OK_AND_ASSIGN(std::unique_ptr<HloModule> module,
                           ParseAndReturnVerifiedModule(hlo_string));
   InsertCopies(module.get());
-  VLOG(2) << module->ToString() << "\n";
```

Comment:
Please move to a separate PR

Reply:
Done: https://github.com/tensorflow/tensorflow/pull/57042